server/build.gradle

```diff
@@ -36,9 +36,10 @@ protobuf {
     generateProtoTasks {
         all().each { final task ->
             task.plugins {
-                grpc {
-//                    option 'nano=true'
-                }
+                grpc {}
```

Comment:
We do. Adding even empty closure in this section adds protoc option --%CLOSURE_NAME%_out. In our case, this empty block is a signal to generate grpc services